test/PriceFeed.t.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import 'forge-std/Test.sol';
+import { SafeCast } from 'openzeppelin-contracts/contracts/utils/math/SafeCast.sol';
+import { PriceFeed } from '../src/PriceFeed.sol';
+import { TestChainlinkAggregator } from './helpers/TestChainlinkAggregator.sol';
+
+contract PriceFeedTest is Test {
+    uint256 constant STALE_AFTER = 42 hours;
+    uint256 constant PRICE_UPPER_BOUND = 0.01e18; // i.e. 100 tokens buy 1 ETH
+    uint256 constant PRICE_LOWER_BOUND = 0.00001e18; // i.e. 100K tokens buy 1 ETH
```

Comment:
These are flipped for a ETH/Token pair, no?

Should be:

```
uint256 constant PRICE_UPPER_BOUND = 100e18; // i.e. 100 tokens buy 1 ETH
uint256 constant PRICE_LOWER_BOUND = 100000e18; // i.e. 100K tokens buy 1 ETH
```

Reply:
Oops!
Missed that when changing the price feed to ETH/Token:

Fixed: https://github.com/nounsDAO/token-buyer/pull/4/commits/d385758a3ba5a895f7ce32a1b82b8953dc3c6719